### ISSUE TITLE
Unsubscribe to all the listener

### ISSUE THREAD

**Issue by chenxeed** [developit/mitt#102]
Hello,

should there be a method to remove all listeners of the `emitter`, to save performance?

I have an example case when we initialize a component, and the component needs to use the event emitter, and pass it to other function to listen to the component event:

```js
// component.js
componentInit() {
  method.init(emitter);
}

// method.js
init (emitter) {
  emitter.on('something', doSomething);
}
```

Then, when the component is about to be destroyed, should the emitter be "cleaned up" so the method won't have to preserve the event listener anymore? For ex:

```js
// component.js
componentDestroy() {
  emitter.off('*');
}
```

**Comment by chenxeed** [comment]
Perhaps this PR is what we need for this issue <3

https://github.com/developit/mitt/pull/72

**Comment by papertokyo** [comment]
I'm implementing mitt to replace a Vue event bus right now, and this is the only missing piece of functionality.

I like `emitter.off('*')` because it's fairly obvious, but `emitter.clear()` is also fine if there's issues with that.

**Comment by developit** [comment]
I'm just comparing the two options before I merge. I will probably go with .clear().

**Comment by developit** [comment]
`.all` shipped in [2.1.0](https://github.com/developit/mitt/releases/tag/2.1.0) which is now out of preview:

```js
const events = mitt();
events.on('foo', () => alert(1));
events.on('bar', () => alert(2));

// remove all event handlers:
events.all.clear();
```
